apps/condo/pages/ticket/[id]/index.tsx

```diff
@@ -207,11 +214,14 @@ const TicketIdPage = () => {
         user: auth.user && auth.user.id,
     }, () => { refetchComments() })
 
-    const { link, organization } = useOrganization()
+    const { link, organization: userOrganization } = useOrganization()
```

Comment:
it's confusing. we have an organisation from props and why do we need the hook?

Reply:
In 1a23bf1 I took out the `TicketPageContent` component, into which I pass the `organization`, `employee` and `TicketContent`.
`TicketContent` is a component with ticket fields (`TicketContent` in contact center also has an `organization` field line and it doesn't seem worth inserting it here)